### PR TITLE
Feature/post migration styling

### PR DESCRIPTION
**Pull request by Alexangelj**
## Description
- Styles topbar
- Styles market header
- Styles Settings modal

### CODE REVIEW

File: app/src/components/Settings/Settings.tsx [hunk 1]
```diff
@@ -26,19 +27,20 @@ export const Settings = () => {
...
-              <h5>Even More User Settings</h5>
-            </Box>
-            <Spacer size="sm" />
-            <Box alignItems="flex-start" justifyContent="space-between" column>
-              <h5>Preferred Stablecoin</h5>
-              <h5>Other Settings</h5>
-              <h5>Even More User Settings</h5>
-            </Box>
-          </Box>
+          <StyledRow>
```
Comment (zachdt) [inline app/src/components/Settings/Settings.tsx:30]: I'll make this functional, still need to add deadline variable.

**Review by zachdt — APPROVED** [review]

File: app/src/components/Market/OptionsTable/OptionsTable.tsx [hunk 2]
```diff
@@ -72,7 +72,14 @@ const OptionsTable: React.FC<OptionsTableProps> = (props) => {
             {options[type].map((option, i) => {
               const { breakEven, price, strike, address } = option
               return (
-                <TableRow key={address}>
+                <TableRow
+                  key={address}
```
Comment (zachdt) [inline app/src/components/Market/OptionsTable/OptionsTable.tsx:76]: In my branches order context, the addItem function includes an orderType string with more options than the boolean 'buy or mint'.  Therefore, clicking the row should open a trade order by using the syntax  'onAddItem(option, 'BUY')'

Comment (zachdt) [reply app/src/components/Market/OptionsTable/OptionsTable.tsx:76]: currently there are BUY SELL MINT EXEC LP and WLP orderTypes

**Review by zachdt — APPROVED** [review]

File: app/src/components/Market/OptionsTable/OptionsTable.tsx (hunk 2 above)
Comment (Alexangelj) [reply app/src/components/Market/OptionsTable/OptionsTable.tsx:76]: okay, ill make a change with that exact syntax? 'onAddItem(option, 'BUY')'

Comment (zachdt) [reply app/src/components/Market/OptionsTable/OptionsTable.tsx:76]: If you want to start the user at the BUY order screen, that is correct

Comment (zachdt) [reply app/src/components/Market/OptionsTable/OptionsTable.tsx:76]: Ill add a enum later

**Pull request merged**